cfpb/cfgov-refresh
# Add Page Preview fields to be shown on the Post Preview Organism

- Changed expandable.html to be a macro and updated affected files
 - The change to the macro was necessary for a future incarnation of the Filterable List Controls

## THIS RELIES ON #1322 
@kave @rosskarchner 

### To Test
- Make sure Page Preview fields save properly. Look for them in the page configuration/settings tab.
- Go to the prototype pages and make sure they're still rendering
- Add an expandable and make sure that renders properly
